internal/cmd/connect/command_cluster.go

```diff
@@ -65,10 +66,12 @@ func (c *clusterCommand) autocompleteConnectors() []string {
 		return nil
 	}
 
+	keys := types.GetSortedKeys(connectors)
```

Comment:
This is to fix a flaky autocomplete test because the connectors are in an unordered map.

I couldn't use `GetSortedValues` here because the values don't satisfy `constraints.Ordered`.